src/objects/job.py

```diff
@@ -19,6 +20,7 @@ def __init__(
         name_safe: str,
         build_id: Optional[str],
         gcs_bucket: str,
+        gcs_creds_file: Optional[str],
```

Comment:
The expected format can also be provided in the docstrings args [Line 36](https://github.com/RedHatQE/firewatch/blob/f1d6c0ba51764597cfc62cb99403d583e16218bc/src/objects/job.py#L36)